### ISSUE TITLE
EventBus callback priority

### ISSUE THREAD

**Issue by johny-b** [golemfactory/golem-core-python#48]
`EventBus.listen(callback, priority=2)`

I'm not sure if this is a good idea. Current usecase:
1. I have database logger that listens to all `NewResource` events and saves new resources to a database
2. I have a separate piece of logic that reacts to a particular `NewResource` event and calculates an aggregate using the database

The problem is: result of the calculation in p.2 depend on whether the new event was already saved in the database (by p.1 logger) or not. Currently this might be this or that way - callbacks are executed in parallel.

"Lower priority" for the callback in p.2 would ensure it is executed only after all callbacks with higher priority executed before.
